docs/man/skuba.1.md

```diff
@@ -39,6 +39,7 @@ reconfiguration in an easy way.
 **skuba-cluster-init**(1),
 **skuba-cluster-status**(1),
 **skuba-cluster-upgrade-plan**(1),
+**skuba-cluster-upgrade-check**(1),
```

Comment:
another small SMALL nit

I think these are supposed to be in alphabetical order.